src/components/Markdown/Tiles.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'src/components/Link';
+import Icon from '@ably/ui/core/Icon';
+import { IconName } from '@ably/ui/core/Icon/types';
+
+export type TileProps = {
+  title: string;
+  description: string;
+  image: IconName;
+  link: string;
+};
+
+export const Tile = ({ title, description, image, link }: TileProps) => {
+  return (
+    <Link to={link} rel="noreferrer" className="block no-underline">
+      <section className="flex items-start gap-4 p-4 bg-neutral-100 rounded-lg border border-neutral-300 transition-colors hover:bg-neutral-200 hover:border-neutral-400 min-h-[106px]">
+        <Icon name={image} size="2rem" />
+        <div className="flex-1 max-w-[264px]">
+          <h3 className="text-[18px] font-bold leading-[120%] tracking-tight mb-2 text-neutral-1300">{title}</h3>
+          <p className="text-sm font-medium leading-[160%] tracking-normal text-neutral-800 max-h-[44px] overflow-hidden">
+            {description}
+          </p>
+        </div>
+      </section>
+    </Link>
+  );
+};
+
+export const Tiles = ({ children }: { children: TileProps[] }) => {
+  return (
+    <div className={'grid grid-cols-1 sm:grid-cols-2 gap-4 pb-40'}>
```

Comment:
This is fine but we wouldn't ordinarily have a string prop inside { }, can use the string directly